api/client/client.go

```diff
@@ -59,8 +59,8 @@ type ListenOption struct {
 
 // ListenEvent is used to send data or error values for Listen
 type ListenEvent struct {
-	action Action
-	err    error
+	Action Action
+	Err    error
```

Comment:
needed to make these accessible so i could just import the client instead of recreate it